docs/deployment/local-deployment.rst

```diff
@@ -142,18 +139,18 @@ If we don't have it already, we need to create a new file ``application.yml``. W
     # ... other configuration
 
     repository:
-        type: 5
-        blazegraph:
-            url: http://blazegraph:8080/blazegraph
+        type: 4
```

Comment:
It would also be interesting to know where the magic number `4` comes from. The repository type is mentioned in the [FDP source][1], but it's not immediately clear to me what that is based on:

```yaml
...
# valid repository type options {1 = inMemoryStore, 2 = NativeStore, 3 = AllegroGraph, 4 = graphDB, 5 = blazegraph}
repository:
  main:
    type: ${FDP_TRIPLE_STORE_TYPE:1}
...
```

[1]: https://github.com/FAIRDataTeam/FAIRDataPoint/blob/d0ca93f0ab993450a53f47bd3fd57270180c24f6/src/main/resources/application.yml#L57

Reply:
I recon the order in which things were implemented a couple of years ago 🤷

Reply:
FYI: the types are defined in [org.fairdatapoint.config.properties.RepositoryConnectionProperties](https://github.com/FAIRDataTeam/FAIRDataPoint/blob/d0ca93f0ab993450a53f47bd3fd57270180c24f6/src/main/java/org/fairdatapoint/config/properties/RepositoryConnectionProperties.java#L34)